containers/opensc-test-piv/Containerfile

```diff
@@ -11,3 +11,4 @@ WORKDIR /src
 RUN /src/.github/setup-linux.sh piv
 RUN /src/.github/build.sh
 CMD /src/.github/test-piv.sh
+CMD /src/.github/test-piv.sh valgrind
```

Comment:
I'm not an expert, but I think only exactly one CMD is executed, i.e. the last CMD will overwrite the previous one

Reply:
Correct. In other cases, I commented out the first one as this is used only for local testing to avoid wasting GH actions minutes. Will fix on the next rebase.